jest.config.json

```diff
@@ -12,7 +12,8 @@
   "clearMocks": true,
   "testMatch": ["<rootDir>/tests/**/*.(test).ts(x)?"],
   "moduleNameMapper": {
-    "\\.(css|scss|less)$": "<rootDir>/tests/__utils__/style-mock.ts"
+    "\\.(css|scss|less)$": "<rootDir>/tests/__utils__/style-mock.ts",
+    "react-markdown": "<rootDir>/node_modules/react-markdown/react-markdown.min.js"
```

Comment:
can you make a comment here why this is needed

Reply:
what are the syntax issues? I tested and build from your branch with just the `moduleNameMapper` change and didn't encounter any

Reply:
I should note that keeping either the`moduleNameMapper` line for `react-markdown`, or using the `transformIgnorePatterns` line works. Both are not needed

![Screenshot 2024-07-08 at 12 07 26 PM](https://github.com/yext/chat-ui-react/assets/170358021/5eccc6a6-99cc-48d8-86ac-6a5f551a08b7)

Reply:
to clarify, my original comment is asking if we can add a comment in the config file

Reply:
Added